frontend/src/components/layout/Navbar/Navbar.tsx

```diff
@@ -55,13 +55,14 @@ export const Navbar: React.FC = () => {
   }, [isOpened]);
 
   return (
-    <Nav>
+    <Nav aria-label="navigation">
```

Comment:
The semantic tag here is enough for accessibility here.  Is there another reason you're adding this?

```suggestion
    <Nav>
```

Reply:
Currently, I am using this label for a `roleMatcher` in the test file.

Reply:
Ahh okay, can we update this to use `data-testid`?  I've done this in the past where I would use aria tags for testing lookups, but I don't believe that's the best solution

Reply:
Done.